### PR TITLE
docs: add new community plugins project area

### PR DESCRIPTION
**Pull request by vinzscam**
## Hey, I just made a Pull Request!

Let's add a new project area for the [Community plugins repository](https://github.com/backstage/community-plugins).

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [x] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

File: OWNERS.md
Comment (benjdlambert) [inline OWNERS.md:115]: I wonder if it makes sense to update the scope for this for now to something a little wider. 

```suggestion
Scope: Tooling and Community Repo Maintainers for the Backstage [Community Plugins repository](https://github.com/backstage/community-plugins)
```

Comment (benjdlambert) [reply OWNERS.md:115]: Should probably link out to some of the governance to what is a Reviewer / Repo Maintainer / Plugin Maintainer in the community docs

**Review by benjdlambert — APPROVED** [review]

File: OWNERS.md
Comment (Rugvip) [inline OWNERS.md:122]: @BethGriggs @kadel @awanlin @tudi2d could each of you approve this PR if this all looks good? 🙏

**Review by tudi2d — APPROVED** [review]

**Review by awanlin — APPROVED** [review]
🚀

**Review by kadel — APPROVED** [review]

**Comment by benjdlambert** [comment]
Wahey :tada: we're all signed off! Welcome!

**Pull request merged**
